### ISSUE TITLE
Reproducing the results in "Multivariate Probabilistic Time Series Forecasting via Conditioned Normalizing Flows" in need of Parameters

### ISSUE THREAD

**Issue by Usama-Samad** [zalandoresearch/pytorch-ts#165]
Hello, 

Im trying to reproduce the results for all the datasets mentioned in the paper, but in examples of pytorch-ts there are only two examples with solar and traffic dataset

I’d really appreciate it if you could provide your hyperparamer settings for all the remaining datasets: **Exchange, Electricity, Taxi, Wikipedia**
Respectively for all the models **GRU-Real NVP, GRU-MAF and Transformer-MAF**

Would really appreciate your help.
